sorts/cocktail_sort.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+
+namespace cocktail_sort
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Please enter some integers, separated by spaces:");
+            string input = Console.ReadLine();
+            string[] integers = input.Split(' ');
+            int[] data = new int[integers.Length];
+            for (int i = 0; i < data.Length; i++)
+                data[i] = int.Parse(integers[i]);
+            
+            Console.Write("\nUnsorted: ");
+            for (int i = 0; i < data.Length; i++)
```

Comment:
You can write the print procedure shorter. [See here](https://stackoverflow.com/questions/16265247/printing-all-contents-of-array-in-c-sharp/16265268)